.github/workflows/test.yml

```diff
@@ -56,9 +56,9 @@ jobs:
             return config
 
           matrix = [
-            {"kernel": "LATEST", "runs_on": ["ubuntu-latest", "self-hosted"], "arch": "x86_64", "toolchain": "gcc"},
-            {"kernel": "LATEST", "runs_on": ["ubuntu-latest", "self-hosted"], "arch": "x86_64", "toolchain": "${{ needs.llvm-toolchain.outputs.llvm }}"},
-            {"kernel": "LATEST", "runs_on": ["z15", "self-hosted"], "arch": "s390x", "toolchain": "gcc"},
+            {"kernel": "LATEST", "runs_on": [], "arch": "x86_64", "toolchain": "gcc"},
```

Comment:
Can we consider introducing constants for the architectures we support instead of duplicating the strings?

Reply:
slightly over-engineered it with enums.